extension/test/integration/make-payment.handler.spec.js

```diff
@@ -0,0 +1,74 @@
+const { expect } = require('chai')
+
+const ctpClientBuilder = require('../../src/ctp/ctp-client')
+const iTSetUp = require('./integration-test-set-up')
+const c = require('../../src/config/constants')
+
+describe('::makePayment::', () => {
+  let ctpClient
+
+  before(async () => {
+    ctpClient = ctpClientBuilder.get()
+    await iTSetUp.initServerAndExtension(ctpClient)
+  })
+
+  afterEach(async () => {
+    await iTSetUp.cleanupResources()
+  })
+
+  it('given a payment ' +
+    'when makePayment custom field is set and response from Adyen is Authorised, ' +
+    'then should set custom field makePaymentResponse, ' +
+    'should add transaction, ' +
```

Comment:
given when then, the idea to explain it with a complete sentence, maybe one should is enough for the naming.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/257/commits/9d94d96be128eda29737ace49382d4e6c32345bd